bigbluebutton-html5/private/config/settings-development.json

```diff
@@ -46,6 +46,9 @@
           "layout": false
         }
       },
+      "branding": {
```

Comment:
Please set (by default) to false. Also please add this to `settings-production.json`